roles/nvidia-gpu-operator/tasks/k8s.yml

```diff
@@ -5,12 +5,16 @@
 # So for now we will run /usr/local/bin/helm commands directly...
 - name: install gpu-operator helm repo
   command: /usr/local/bin/helm repo add nvidia "{{ gpu_operator_helm_repo }}"
+  changed_when: false
 
 - name: update helm repos
   command: /usr/local/bin/helm repo update
+  changed_when: false
 
 - name: Remove existing GPU Operator labels to accommodate potenial configuration changes
-  shell: kubectl label node --all nvidia.com/gpu.deploy.container-toolkit-  nvidia.com/gpu.deploy.dcgm- nvidia.com/gpu.deploy.dcgm-exporter- nvidia.com/gpu.deploy.device-plugin- nvidia.com/gpu.deploy.driver- nvidia.com/gpu.deploy.gpu-feature-discovery- nvidia.com/gpu.deploy.node-status-exporter- nvidia.com/gpu.deploy.operator-validator-
```

Comment:
I just added this line and if I remember there was an edge-case error that required me to use shell here instead of command. Let me double check that this is passing the tests.

Reply:
Looks like no.